codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/integration/Waiters.java

```diff
@@ -58,8 +58,16 @@ public Set<Symbol> getAdditionalClientOptions() {
         return emptySet();
     }
 
+    public boolean enabledForService(Model model, ShapeId service) {
```

Comment:
Why do we return true on both waiters? I see that on aws/aws-sdk-go-v2/#2717 we use a public member on waiters2 instead of relying on this interface

Reply:
The default is true, SDK v2 is extending both of these hooks to be conditional.

Reply:
ah, I see, `AwsWaiters` extends from this class and it defines whether it should use this or not.